test/testUtils.ts

```diff
@@ -141,6 +141,10 @@ export async function setTimestamp(timestamp: number): Promise<any> {
   return ethers.provider.send("evm_mine", [timestamp])
 }
 
+export async function forceAdvanceOneBlock(): Promise<any> {
+  return ethers.provider.send("evm_mine", [])
+}
+
```

Comment:
should probably refactor all these duplicate calls:

![Screen Shot 2021-03-08 at 1 30 55 PM](https://user-images.githubusercontent.com/972382/110384507-95726180-8012-11eb-896c-c97c550f7837.png)